src/AppInstallerCLICore/Core.cpp

```diff
@@ -78,6 +79,13 @@ namespace AppInstaller::CLI
         // The root command is our fallback in the event of very bad or very little input
         std::unique_ptr<Command> command = std::make_unique<RootCommand>();
 
+        // Load the settings file and warn if necessary.
+        auto warnings = Settings::UserSettings::Instance().GetWarnings();
```

Comment:
>Settings::UserSettings::Instance() [](start = 24, length = 34)

I would suggest a standalone function that returns this, such as:

Settings::User()

Or similar.  I really hate cluttered call sites.

Reply:
Also, this could just be directly in the for init.

---
In reply to: [442408327](https://github.com/Microsoft/pkg/pull/436#discussion_r442408327) [](ancestors = 442408327)